test/e2e/operator_groups_e2e_test.go

```diff
@@ -426,8 +426,8 @@ var _ = Describe("Operator Group", func() {
 		// Create crd so csv succeeds
 		// Ensure clusterroles created and aggregated for access provided APIs
 
-		// Generate namespaceA
```

Comment:
If you're going to have phases of a test, use a logger to make them appear in the output to make debugging failures easier.